examples/src/main/scala/mhtml/examples/HelloWorld.scala

```diff
@@ -1,4 +1,5 @@
 package mhtml.examples
+import mhtml._
```

Comment:
Uh this is bad, it means that first experience with the lib is going to be `error: could not find implicit value for parameter ev: XmlAttributeEmbeddable[String]`, which cannot really be made nice because `@implicitNotFound` needs to be defined locally. Maybe we should forget about the idea of splitting `xml-light` appart... WDYT? Sorry for the back and forth...